internal/packages/archetype/package_test.go

```diff
@@ -63,15 +62,6 @@ func createPackageDescriptorForTest(packageType, kibanaVersion string) PackageDe
 	if err != nil {
 		panic(err)
 	}
-	if packageType == "content" {
-		minSpecVersion := semver.MustParse("3.4.0")
-		if !specVersion.LessThan(minSpecVersion) {
-			panic("this code can be removed")
-		}
-		version = "0.1.0"
-		specVersion = *minSpecVersion
-		excludeChecks = append(excludeChecks, "PSR00002")
-	}
```

Comment:
In principle this code is not needed anymore because content packages are already GA cc @mrodm.